compiler/qsc_partial_eval/src/lib.rs

```diff
@@ -1310,6 +1310,10 @@ impl<'a> PartialEvaluator<'a> {
         args_span: PackageSpan,        // For diagnostic purposes only.
         callee_expr_span: PackageSpan, // For diagnostic puprposes only.
     ) -> Result<Value, Error> {
+        if matches!(callable_decl.kind, qsc_fir::fir::CallableKind::Measurement) {
```

Comment:
Do you plan to update the standard library in a later PR to use `@Measurement` and remove the special cases for measuring qubits?

Reply:
Yes, that's the plan.